### ISSUE TITLE
S1450 not appearing in VS2015 IDE

### ISSUE THREAD

**Issue by duncanp-sonar** [SonarSource/sonar-csharp#1449]
### Description

S1450 issues are not appearing in the VS2015 IDE.

### Repro steps
```
namespace S1450_FalseNegativeInVS2015IDE
{
    public class Example1
    {
        int field; // << expecting S1450 to be raised here but does not appear in VS2015.3 IDE
        // It is correctly detected when running from the command line and in the VS2017 IDE

        public void Increment(int val)
        {
            field = val + 1;
        }
    }
}
```

### Expected behavior

Please provide a description of the behavior you expect.

### Actual behavior

The issue does not appear in the IDE, but it does appear when analysing during builds.

### Known workarounds

Works as expected in VS2017.

### Related information

* SonarC# Version: 7.0+
* Visual Studio Version: VS2015 Update 3

###Note
#1448 is a separate issue about a genuine false negative in the rule when using expression method bodies. This issue is about the diagnostic not appearing in the VS2015 IDE.

**Comment by valhristov** [comment]
Hi @duncanp-sonar, do you keep the project you used to reproduce the issue? I cannot reproduce it locally using the latest nuget:

![image](https://user-images.githubusercontent.com/23195959/43463066-00a088e8-94d8-11e8-9430-6cef9b83a444.png)

Could this be caused by having "test" in the project name as I suggested in #1536?

**Comment by Evangelink** [comment]
@valhristov could you check whether your rule rewrite also fixes this issue?

**Comment by valhristov** [comment]
It takes a couple of seconds for the issues to appear (not only S1450) but they appear consistently without the need to rebuild.
